### PR TITLE
blog: Add 2021-07-02 weekly report

### PR DESCRIPTION
**Pull request by Prnyself**
Signed-off-by: Prnyself <281239768@qq.com>

### CODE REVIEW

File: blog/2021-07-02-weekly-report.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+For more details, please refer to [issue: Add Copier and Mover tests](https://github.com/beyondstorage/go-integration-test/issues/26).
+
+## Community
+
+### Website
+
+Our new website is now deployed and online! Please check <https://beyondstorage.io> to take a peer, 
+and we are now keeping adding more information into the website. Please feel free to make suggestion by contacting us.
+Or you can also [commit an issue](https://github.com/beyondstorage/site/issues) for us, both ways are welcome.
```
Comment (Xuanwo) [inline blog/2021-07-02-weekly-report.md:71]:
```suggestion
Or you can also [submit an issue](https://github.com/beyondstorage/site/issues) for us, both ways are welcome.
```

File: blog/2021-07-02-weekly-report.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+### Integration Test
+
+With the development of new features in [go-storage] and go-service-*, the integration test cases should be completed.
+
+This week, [@JinnyYi] commits test cases for `Copier` and `Mover`, which enriched our test cases. Nicely done!
+
+For more details, please refer to [issue: Add Copier and Mover tests](https://github.com/beyondstorage/go-integration-test/issues/26).
+
+## Community
+
```
Comment (JinnyYi) [inline blog/2021-07-02-weekly-report.md:66]: Is it necessary to introduce [GSP-128: Community Organization](https://github.com/beyondstorage/specs/blob/master/rfcs/128-community-organization.md)?

File: blog/2021-07-02-weekly-report.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+### Website
+
+Our new website is now deployed and online! Please check <https://beyondstorage.io> to take a peer,
```
Comment (xxchan) [inline blog/2021-07-02-weekly-report.md:69]: Did you want to say "take a look"?

Comment (Prnyself) [reply blog/2021-07-02-weekly-report.md:69]: I'd like to say "take a glance"

File: blog/2021-07-02-weekly-report.md (hunk 2 above)
Comment (Prnyself) [reply blog/2021-07-02-weekly-report.md:66]: Yes, I will add this part.

Comment (Prnyself) [reply blog/2021-07-02-weekly-report.md:66]: Content has been added, please start another review.

**Review by xxchan — APPROVED** [review]

**Pull request merged**